webpack-watcher.js

```diff
@@ -0,0 +1,57 @@
+const childProcess = require('child_process');
+
+module.exports = (options) => {
+   console.log("Forking " + options.username);
+   const child = childProcess.fork(__dirname + '/webpack-compiler.js')
+
+   let bundledCallback = () => {}
```

Comment:
I find this name a little unintuitive because my first reading is to treat "bundled" as an adjective, i.e., a callback that is somehow "bundled". Since it's effectively an event handler, I'd suggest something like `onBundled`.